doc/api/crypto.md

```diff
@@ -172,14 +172,14 @@ console.log(encrypted);
 // Prints: ca981be48e90867604588e75d04feabb63cc007a8f8ad89b10616ed84d815504
 ```
 
-### cipher.final([output_encoding])
+### cipher.final([outputEncoding])
```

Comment:
Either that, or check all the links and update them, its easy to grep for what they used to be, though a bit painful to fix them all. Still, consistent camelCase would improve the docs, if you've the appetite for it.